app/src/interfaces/input-rich-text-html/get-editor-styles.ts

```diff
@@ -140,12 +140,12 @@ blockquote {
 video,
 img {
 	max-width: 100%;
-	border-radius: ${cssVar('--border-radius')};
+	border-radius: ${cssVar('--theme--border-radius')};
 	height: auto;
 }
 iframe {
 	max-width: 100%;
-	border-radius: ${cssVar('--border-radius')};
+	border-radius: ${cssVar('--theme--border-radius')};
 }
 hr {
 	background-color: ${cssVar('--border-normal')};
```

Comment:
should `--border-normal` be updated? As it seems like this file contains the last 3 `--border-normal` from a quick global search.